### DISCUSSION TITLE
Solved: Astroid Template One 2.6.4: How to get rid of shadows on articles?

### DISCUSSION THREAD

**Discussion by heikowtb** [templaza/astroid-framework#315] [Q&A]
Hi,

how can i get rid of the shadows around the articles? I think I tried every class I could find, but no matter to which class I add box-shadow: none, nothing happens.

Thank you for your help.
Regards,
Heiko

**Comment by Chacapamac** [comment]
I @heikowtb 

Try 
```
.shadow, .card-layout, .article-wraper-inner {
box-shadow: none !important; <!-- Take out shadow -->
border: 2px #ddd solid !important; <!-- Add a cleaner simple border -->
}
```

You can easily discover css of elements by using your **html page inspector** that are part of all modern browsers.

![Screen Shot 2023-04-12 at 10 27 57 AM](https://user-images.githubusercontent.com/1907276/231489480-b17a6f69-1587-40e9-9884-481d9fc9d1e8.jpg)

**You can also be more specific with**
```
div.article-wraper-inner {
  box-shadow: none !important;
}

```

**Reply by heikowtb** [reply]
That's it, tank you!
Regards,
Heiko

**Reply by Chacapamac** [reply]
Good Day!
